modules/cluster-cloud-controller-config-osp.adoc

```diff
@@ -0,0 +1,328 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-cloud-config-reference.adoc
+
+:_content-type: REFERENCE
+[id="cluster-cloud-controller-config_{context}"]
+= The {rh-openstack} Cloud Controller Manager (CCM) config map
+
+A {rh-openstack-first} CCM config map defines how your cluster interacts with your {rh-openstack} cloud. By default, this configuration is stored under the `cloud.conf` key in the `cloud-conf` config map in the `openshift-cloud-controller-manager` namespace. 
+
+[IMPORTANT]
+====
+The `cloud-conf` config map is generated from the `cloud-provider-config` config map in the `openshift-config` namespace.
+
+To change the settings that are described by the `cloud-conf` config map, modify the `cloud-provider-config` config map. 
+
+As part of this synchronization, the CCM Operator overrides some options. For more information, see "The {rh-openstack} Cloud Controller Manager".
```

Comment:
Where can we find the "The {rh-openstack} Cloud Controller Manager" reference? should it be a link?

Reply:
I have to merge #53068 prior to this PR going live. Link will be included in this document.

Reply:
This comes from #52520. I can add a link after that's merged.